pinc/links.inc

```diff
@@ -78,3 +78,46 @@ function private_message_link($proofer_username, $target = "_blank")
 
     return $a;
 }
+
+// -----------------------------------------------------------------------------
+
+function return_to_project_page_link($projectid, $url_suffix = "")
+// Returns a string containing a snippet of HTML to return to a project page.
+// Optional second argument could be page anchor, e.g "#holds"
+// or additional URL parameter, e.g. "&detail_level=4"
+{
+    global $code_url;
+
+    $url_suffix = html_safe($url_suffix);
+    return sprintf(
+             _("Return to the <a %s>project page</a>"),
+             "href='$code_url/project.php?id=$projectid$url_suffix' target='_top'"
+           );
+}
+
+// -----------------------------------------------------------------------------
+
+function return_to_round_page_link($round_id)
```

Comment:
probably not just an issue here, but we use `projectid` above, but `round_id` here. Ideally they'd be the same casing strategy I imagine.

Reply:
You're right - there is inconsistency in the existing code. I chose to keep underscores, since otherwise the query parameter would read `?round_id=$roundid`, so I changed the parameter of the project page functions to `project_id`. The variable `$projectid` is used in many other places though, so difficult to find a perfect solution.

Reply:
I appreciate that we are inconsistent between `$projectid` and `$round_id`, but we're pretty consistent in those two forms being used throughput the code and suggest we stick with `$projectid` in the functions above.